Datasnap/Classes/Utilities/Datasnap.m

```diff
@@ -172,21 +185,20 @@ - (void)onDataInitialized
 - (void)trackEvent:(BaseEvent*)event
 {
     event.dataSnapVersion = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
-    NSDateFormatter* dateFormatter = [[NSDateFormatter alloc] init];
+    NSDateFormatter* dateFormatter = [NSDateFormatter new];
     [dateFormatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ssZZZZZ"];
     event.created = [NSString stringWithFormat:@"%@", [dateFormatter stringFromDate:[NSDate date]]];
     event.deviceInfo = self.deviceInfo;
     event.user = self.user;
-    event.organizationIds = [[NSMutableArray alloc] init];
-    event.projectIds = [[NSMutableArray alloc] init];
-    [event.organizationIds addObject:self.organizationId];
-    [event.projectIds addObject:self.projectId];
+    event.organizationIds = @[ self.organizationId ];
```

Comment:
Not keen on this.  I would prefer an init handler in BaseEvent that would initialize properties of the event as empty arrays.  What if there are different areas of the app that add something to this array?  Hard coding the array like this seems dangerous.